solutions/cron/vercel.json

```diff
@@ -0,0 +1,24 @@
+{
+  "crons": [
+    {
+      "path": "/api/cron/1m",
```

Comment:
On the demo it looks like every minute and 10 minutes had the same story, is that a fluke?

Reply:
I thought this too until I hit refresh and then it diverged. Problem with `getStaticProps`?

Reply:
It can happen when the 1min and 10min cron intersect, but usually the 10min one should lag a bit behind the 1min one:

![CleanShot 2023-02-21 at 17 56 14](https://user-images.githubusercontent.com/28986134/220485610-6078fb83-48c6-4021-99aa-9d7bf11ae196.png)

Reply:
@manovotny oh good catch, I might wanna do res.revalidate instead...

Reply:
refactored to using SWR to make sure data is always up to date <picture data-single-emoji=":checked:" title=":checked:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..sSCB6QmsGaXsLn23.j2-aAwgJ85bLYu9EvU0KapopJLIeZYOy9NZnXAk-O6cHOp5tXoHd9MNjY4pQD92LGBBvuSM7Ywob5EG-4MyUDu6IMZzcub3utbxkM2dUd9QZyPdktS1jscxmOA.xPyJeE9omUHLL1eC-WrXNQ" alt=":checked:" align="absmiddle"></picture>

Reply:
This likely happened as the daily write limit was reached. Daily limit is 480, but writing every minute leads to 1440 writes.

We shouldn’t use such a low number, as the rate limit is per account so this also breaks other edge config demos once the limit is reached